bundles/io/org.eclipse.smarthome.io.transport.mqtt/src/main/java/org/eclipse/smarthome/io/transport/mqtt/MqttService.java

```diff
@@ -5,78 +5,51 @@
  * information regarding copyright ownership.
  *
  * This program and the accompanying materials are made available under the
- * terms of the Eclipse Public License 2.0 which is available at
+ * terms of the Eclipse License 2.0 which is available at
```

Comment:
what happened here? Travis liked the version including `Public ` better.

Reply:
I replaced all "public " occurrences with an empty string 😓